### PR TITLE
Adds custom issue types

### LINKED ISSUE #31: Issue type support
Any thoughts on supporting custom issue types? Referring to a security vulnerability as a bug seems odd to me, and I would like to report these tickets as something other than such.

### PR DESCRIPTION
**Pull request by ghost**
- Fixes Pipfile
- resolves https://github.com/github/ghas-jira-integration/issues/31
- adds vscode .devcontainer directory to .gitignore

### CODE REVIEW

**Comment by cmboling** [comment]
Giving it a glance and this PR is 🔥  @AWNCypher! I'm hoping to test this this weekend. Do you mind making changes to the README as well (showing examples other than "Bug")?

**Comment by cmboling** [comment]
@AWNCypher looks good so far, but this seems like a breaking change. How many repos are using this? We may need to tag this as v2

@zacharyr-qb for 👀

File: sync.py [hunk 1]
```diff
@@ -91,11 +92,13 @@ def sync(self, alert, issues, in_direction):
             # alerts cannot be transitioned to "open"
             issue.adjust_state(alert.get_state())
             issue.persist_labels(self.labels)
+            issue.perist_issue_type(self.issuetype)
```
Comment (cmboling) [inline sync.py:95]:
```suggestion
            issue.persist_issue_type(self.issuetype)
```

**Comment by zacharyr-qb** [comment]
I'll give this a try soon.

**Comment by cmboling** [comment]
@zacharyr-qb hey there! were you able to test this in your environment?

**Comment by brentufkes** [comment]
> I'll give this a try soon.

Any chance this will be merged?  Would love support for this

**Comment by cironera** [comment]
Bump - yes - need this feature badly :+1:

**Review by Ricxell — APPROVED** [review]

**Review by syahdany99-beep — APPROVED** [review]
https://github.com/orgs/github/projects/16712/views/1

**Pull request closed without merging**